docs-v2/reference/sdks/node.mdx

```diff
@@ -458,14 +461,26 @@ await nango.setMetadata('<INTEGRATION-ID>', 'CONNECTION-ID', { 'CUSTOM_KEY1': 'C
 
 **Response**
 
-Empty response.
+<Expandable>
+```json
+{
+    "connection_id": "<string | [string]>",
```

Comment:
(unless the Response field above is wrong)

Reply:
The response field is for the actual argument into the node sdk client which makes sense to have camel case, so `connection_id` is correct here since that is what the HTTP call returns